src/ontology/foodon-edit.owl

```diff
@@ -58457,8 +58469,10 @@ SubClassOf(obo:FOODON_03480047 obo:FOODON_03480029)
 
 # Class: obo:FOODON_03490100 (food packaging)
 
-AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "Type of container or wrapping defined by the main container material, the container form, and the material of the liner lids or ends. Also type of container or wrapping by form; prefer description by material first, then by form.  [FDA CFSAN 1995]"@en)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03490100 obo:IAO_0000122)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03490100 "A food contact material which encloses the food and can be in the type of a container, packaging or wrapping."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:FOODON_03490100 "Existing definition was a legacy definition, not ontological and has been moved to comments."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:FOODON_03490100 "https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/food-packaging")
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03490100 "http://langual.org")
```

Comment:
Add yourself to the edit history with a `term editor` annotation (I picked this line arbitrarily).

```suggestion
AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03490100 "http://langual.org")
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03490100 <http://orcid.org/0000-0001-8008-8249>)
```

Reply:
Resolved in later commits.